### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by glenmich** [Blueforcer/awtrix-light#105]
# Bug report

I have installed the V0.60 software on both of my Ulanzi clocks and since then the display keeps blacking after boot.

## Describe the bug

After installing the V0.60 firmware on a new clock and updating it to an other clock via ota-update, none of them are working anymore. If i reboot them, they'll show the awtrix logo, ip address and the time for a blink of an eye but then the display turns black and doesn't react to anything (mqtt-notify texts, or any button on the clock itself).
However i can still change settings via the browser and the device keeps logged in in my network and home assistant.

## Additional information

- Devices involved:
  - Model: Ulanzi Awtrix Smart Pixel Clock 2882 (TC001)
  - AWTRIX-LIGHT version: [ eg. v0.60 ]

## To Reproduce

Boot the clock

## Expected behavior

Cycle trough the menus after booting

## Logs

```text
(optional) Add relevant logs which could help tackle the problem.
```

**Comment by Blueforcer** [comment]
Never heard of this problem.
I assume the brightness was set to 0, or simply switched off with an automation.
make sure you down send mqtt messages to brightness or power with retained option

**Comment by glenmich** [comment]
![image](https://user-images.githubusercontent.com/67523963/235984490-b19b9a46-8974-45dc-bf82-2baa93798a97.png)![image](https://user-images.githubusercontent.com/67523963/235984517-4b01099a-bfe5-47d8-aa0f-e1508ee18064.png)

I did some restarts and got no pixel to light up at all. Before updating to V0.60 everything worked flawless.

I'll try to reinstall the older firmware now again i guess

**Comment by Blueforcer** [comment]
In your screenshot, matrix is switched off.

**Comment by glenmich** [comment]
Yes, i had switched it on and of a couple of times. ![image](https://user-images.githubusercontent.com/67523963/235994183-d7ac1e4e-04c5-4341-9cac-f68382226afb.png)

It seems a bit strange to me that matrix does not turn blue when i toggle the switch. Is this normal?

It's definitely not normal, that the sensors output 0 lux and 0 degree while the box is placed in front of a window in my office

**Comment by glenmich** [comment]
![image](https://user-images.githubusercontent.com/67523963/235995289-e624a390-3f67-4cc7-be4e-72142f7e6bb2.png)

**Comment by Blueforcer** [comment]
The bulb is not lit. Click on it and check the brightness.

What firmware have you flashed? Since 0.60 there are two different versions for different hardware

**Comment by glenmich** [comment]
![image](https://user-images.githubusercontent.com/67523963/235997101-2b70d651-3abd-4326-98b4-31ecfcd14398.png)
It was turned on at 47% trough the auto brightness but switching to manual and setting 100% white/color did not help either.

That's a nice tip! I have downloaded both and don't know anymore which one i took when updating. Since the interface works perfectly fine i'll try to update the other one :)

**Comment by Blueforcer** [comment]
If you're using the ulanzi clock, flash the ulanzi_firmware.bib

**Comment by glenmich** [comment]
Well, that was an easy fix.
Now everything works just fine again!

I am sorry to bother you with my stupidity and thank you for the great work you did ! 😊
